<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>641</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>641</width>
     <height>181</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>29</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>86</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>28</number>
   </attribute>
   <column>
    <property name="text">
     <string>房间ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>创建者ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>玩家人数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>房间状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>玩家1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>玩家2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>玩家3</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>房间总数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>玩家总数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>21</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangG">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>改变g值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>28</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>g值：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnListen">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>19</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动服务</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>20</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>30628</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtToSend">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>539</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>539</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtGet">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>btnChangG</zorder>
  <zorder>textBrowser</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit</zorder>
  <zorder>btnListen</zorder>
  <zorder>txtPort</zorder>
  <zorder>txtToSend</zorder>
  <zorder>btnSend</zorder>
  <zorder>btnClear</zorder>
  <zorder>txtGet</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
